projects/plugins/protect/src/js/routes/scan/scan-admin-section-hero.tsx

```diff
@@ -119,34 +124,49 @@ const ScanAdminSectionHero: React.FC = () => {
 					</AdminSectionHero.Heading>
 					<AdminSectionHero.Subheading>
 						<>
-							<Text className={ styles[ 'subheading-text' ] }>
-								{ __(
-									'We actively review your sites files line-by-line to identify threats and vulnerabilities.',
-									'jetpack-protect'
-								) }
-							</Text>
-							{ fixableList.length > 0 && (
+							{ hasPlan ? (
+								<Text>
+									{ __(
+										'We actively review your sites files line-by-line to identify threats and vulnerabilities.',
+										'jetpack-protect'
+									) }
+								</Text>
+							) : (
 								<>
-									<Button
-										className={ styles[ 'auto-fixers' ] }
-										ref={ setShowAutoFixersPopoverAnchor }
-										variant="primary"
-										weight="regular"
-										onClick={ handleShowAutoFixersClick( fixableList ) }
-									>
+									<Text mb={ 4 }>
 										{ sprintf(
-											/* translators: Translates to Show auto fixers $s: Number of fixable threats. */
-											__( 'Show auto fixers (%s)', 'jetpack-protect' ),
-											fixableList.length
+											// translators: placeholder is the number of total vulnerabilities i.e. "22,000".
+											__(
+												'Every day we check your plugin, theme, and WordPress versions against our %s listed vulnerabilities powered by WPScan, an Automattic brand.',
+												'jetpack-protect'
+											),
+											totalVulnerabilitiesFormatted
 										) }
+									</Text>
+									<Button onClick={ getScan }>
+										{ __( 'Upgrade to unlock malware scanning', 'jetpack-protect' ) }
 									</Button>
-									{ ! scanning && (
-										<OnboardingPopover
-											id="paid-fix-all-threats"
-											position={ isSm ? 'bottom right' : 'middle left' }
-											anchor={ showAutoFixersPopoverAnchor }
-										/>
-									) }
+								</>
+							) }
+							{ fixableList.length > 0 && (
+								<>
+									<div ref={ setShowAutoFixersPopoverAnchor }>
+										<Button
+											className={ styles[ 'auto-fixers' ] }
+											onClick={ handleShowAutoFixersClick( fixableList ) }
```

Comment:
This action appears to cause a white screen now, the console error is not very specific but my impression is that `fixableList` might not be properly defined anymore.